libsolidity/ast/Types.cpp

```diff
@@ -2308,9 +2310,10 @@ MemberList::MemberMap FunctionType::nativeMembers(ContractDefinition const*) con
 	case Location::Bare:
 	case Location::BareCallCode:
 	case Location::BareDelegateCall:
+	case Location::Transfer:
 	{
 		MemberList::MemberMap members;
-		if (m_location != Location::BareDelegateCall && m_location != Location::DelegateCall)
+		if (m_location != Location::BareDelegateCall && m_location != Location::DelegateCall && m_location != Location::Transfer)
```

Comment:
I think about transfer as the equivalent of an external value transfer transaction. There one can provide the gas and the value.